### ISSUE TITLE
Can not launch workflow after editing files in docker

### ISSUE THREAD

**Issue by robkooper** [PecanProject/pecan#2587]
The scripts used to launch the workflow using rabbitmq is only enabled if you don't edit the files. If you edit the files it will end up in a state where it can not launch the job correctly.

**Comment by robkooper** [comment]
See also discussion by @infotroph and @istfer at https://pecanproject.slack.com/archives/C8RD9BRLK/p1586797775042100

**Comment by infotroph** [comment]
Pasting the Slack discussion here for posterity:

[infotoph] Here’s a weird one: just discovered my Dockerized dev stack was failing all runs with rabbitmq.out reporting b'/bin/sh: 1: /usr/bin/R: not found\n'. editing config.php in the web container from $Rbinary="/usr/bin/R" to $Rbinary="/usr/local/bin/R" makes it run successfully, but that’s not something that should have changed recently in the code nor the containers. Anyone have ideas what I broke to cause this? Does $Rbinary get updated from any DB entries or env vars during the run?

[istfer] oh I just got this too

[infotroph]
You just rebuilt your whole stack fresh, correct? Did this show up right away? I’ve been thinking on it and still don’t see what would have changed.

[istfer]
\> You just rebuilt your whole stack fresh
yes, but I have to admit I'm not sure what it was like before I rebuilt. I just applied your fix and it worked

[kooper]
did this happen if you picked edit option? I ran into the same one as well.
